.github/workflows/xunit-tests.yml

```diff
@@ -16,15 +10,12 @@ concurrency:
 ##  chmod - whether the Z3 download needs execute permissions added
 ##  coverage -- unused?
```

Comment:
Is this still unused?

Reply:
Actually I see it is now set in the matrix. but I can't see where it is used. It appears that coverage testing is run on every platform.

Reply:
Yeah, that was there before, and unused, and when I added coverage testing I made it unconditional (which doesn't seem to be a problem). I'll remove that parameter.